enterprise/server/remote_execution/persistentworker/persistentworker.go

```diff
@@ -153,19 +176,27 @@ func (w *Worker) Exec(ctx context.Context, command *repb.Command) *interfaces.Co
 		Inputs:    inputs,
 		Arguments: expandedArguments,
 	}
+
+	log.CtxInfof(ctx, "Sending persistent work request: %+v", req)
```

Comment:
Should these and others be DebugF? Seems like these would be noisy given the volume of persistent work requests

Reply:
Got rid of these - in hindsight I could've looked at a goroutine dump to get this info